backend/requirements.txt

```diff
@@ -1,5 +1,6 @@
-fastapi
-uvicorn
-python-jose[cryptography]  # for JWT tokens
-passlib[bcrypt]  # for password hashing
-pydantic  # already included with FastAPI but making it explicit
+fastapi==0.115.*
```

Comment:
you use == to pin a specific version (usually with .* to allow the patch version to be updated). the versioning scheme is called semantic versioning, the format is major.minor.patch